iam.tf

```diff
@@ -16,7 +16,7 @@ data "aws_iam_policy_document" "sns_feedback" {
 
     principals {
       type        = "Service"
-      identifiers = ["sns.amazonaws.com"]
+      identifiers = ["sns.${data.aws_partition.current.dns_suffix}"]
```

Comment:
is this warranted?

Reply:
This is going to allow the module to be used for other partitions like china or gov cloud.
It doesn't harm :)